README.md

```diff
@@ -49,90 +37,84 @@ app.use(errorHandler({
 app.listen(3000);
 ```
 
-In LoopBack applications, add the following entry to your
-`server/middleware.json` file.
+In LoopBack applications, add the following entry to `server/middleware.json`:
 
 ```json
 {
   "final:after": {
     "strong-error-handler": {
       "params": {
          "debug": false,
-         "log": true,
+         "log": true
        }
     }
   }
 }
 ```
 
-## Content Type
+The above configuration will log errors to the server console, but not return stack traces in HTTP responses.
+For details on configuration options, see below.
 
-Depending on the request header's `Accepts`, response will be returned in
- the corresponding content-type, current supported types include:
-- JSON (`json`/`application/json`)
-- HTML (`html`/`text/html`)
+### Response format and content type
 
-*There are plans to support other formats such as Text and XML.*
+The `strong-error-handler` package supports HTML and JSON responses:
 
-## Options
-
-#### debug
-
-`boolean`, defaults to `false`.
+- When the object is a standard Error object, it returns the string provided by the stack property in HTML/text
+  responses.
+- When the object is a non-Error object, it returns the result of `util.inspect` in HTML/text responses.
+- For JSON responses, the result is an object with all enumerable properties from the object in the response.
 
-If you need to set the environment to development mode, you will need to change the value to `true`.
+The content type of the response depends on the request's `Accepts` header.
 
-When enabled, HTTP responses include all error properties, including
-sensitive data such as file paths, URLs and stack traces. *See Examples below*
+-  For Accepts header `json` or `application/json`, the response content type is JSON.
+-  For Accepts header `html` or `text/html`, the response content type is HTML.
 
-#### log
+*There are plans to support other formats such as Text and XML.*
 
-`boolean`, defaults to `true`.
+## Options
 
-When enabled, all errors are printed via `console.error`. That includes an array of fields (custom error properties)
- that are safe to include in response messages (both 4xx and 5xx).
+| Option | Type | Default | Description |
```

Comment:
> On the other hand, most packages on npm use the narrative format...

I don't think that's true.  There are certainly packages that use both, but the following use tables for option/parameter reference:
- Express [not surprising since I own this doc ;-) ] e.g. http://expressjs.com/en/4x/api.html#express 
- AngularJS e.g. https://docs.angularjs.org/api/ng/function/angular.bind
- Sails: e.g. http://sailsjs.com/documentation/reference/waterline-orm/queries/exec
- http://sailsjs.com/documentation/reference/blueprint-api/add-to
MongoDB e.g. https://docs.mongodb.com/manual/reference/method/db.collection.find/

Others: Blueprint, KeystoneJS, etc. etc.

To be clear, I'm lumping API reference doc for method parameters together with doc for JSON options here, but the two are similar enough that it's apt.

There are of course MANY Node packages that **do** use narrative formatting as well, so I don't think one can claim either is an overwhelming standard. 

In any case, LoopBack docs (both on loopback.io and apidocs.strongloop.com) use tables for this type of reference material, so I don't see why this should be an exception.

>  I have personally run into this issue many times, but never bothered to report it....

Well that's good to know, and it's simple enough to add an anchor in a table row for this purpose as needed (I just tested this and it works).  In this particular case, I think it's moot since there are only two options, so linking to the section head is virtually the same as linking to the table row.